README.md

```diff
@@ -263,7 +347,7 @@ a lot of manually assigned topic-partitions and want to keep only one kafka cons
 
 You can create reusable consumer actor reference like this:
 
-```csharp
```

Comment:
This was right before, wasn't it?

Reply:
Its interchangeable, i'm just making it more consistent